.github/workflows/memsub-promotions-lambdas.yml

```diff
@@ -81,7 +81,8 @@ jobs:
                   functionNames:
                     - MembershipSub-Promotions-to-PromoCode-View-
                     - MembershipSub-PromoCode-View-Dynamo-to-Salesforce-
-                    - MembershipSub-Promotions-Scheduled-Dynamo-Backup-
+                dependencies:
```

Comment:
I noticed that this was missing; I don't think it's strictly necessary for this PR to work.